v2/AdminCore.Services/EventService.cs

```diff
@@ -366,14 +363,14 @@ private EventDto ValidateRemainingHolidaysAndCreate(Event newEvent)
       throw new Exception("Not enough holidays to book");
     }
 
-    private Event BuildNewEvent(int employeeId, EventTypes eventTypes)
+    private Event BuildNewEvent(int employeeId)
```

Comment:
This overrides a previous fix, revert this

Reply:
Fixed